### PR TITLE
[Backport Foxy] Update QD

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

**Comment by dabonnie** [comment]
Once the DCO passes this looks like it's ready (unless still in progress?).

File: QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -1,195 +1,195 @@
...
+
+The `statistics_msgs` package contains ROS 2 message definitions for reporting statistics for topics and system resources.
+
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/foxy/statistics_msgs/QUALITY_DECLARATION.md).
+
+#### `std_msgs`
+
+The `std_msgs` package provides many basic message types.
+
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/common_interfaces/blob/foxy/std_msgs/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:172]:
```suggestion
It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/common_interfaces/blob/foxy/std_msgs/QUALITY_DECLARATION.md).
```

File: QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -1,195 +1,195 @@
...
+
+The `rosidl_default_runtime` provides CMake functionality for finding and adding runtime dependencies for rosidl packages.
+
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/rosidl_defaults/blob/foxy/rosidl_default_runtime/QUALITY_DECLARATION.md).
+
+#### `statistics_msgs`
+
+The `statistics_msgs` package contains ROS 2 message definitions for reporting statistics for topics and system resources.
+
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/foxy/statistics_msgs/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:166]:
```suggestion
It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/foxy/statistics_msgs/QUALITY_DECLARATION.md).
```

File: QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -1,195 +1,195 @@
...
+
+The `rcpputils` package is a C++ API consisting of macros, functions, and data structures intended for use throughout the ROS 2 codebase.
+
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcpputils/blob/foxy/QUALITY_DECLARATION.md).
+
+#### `rosidl_default_runtime`
+
+The `rosidl_default_runtime` provides CMake functionality for finding and adding runtime dependencies for rosidl packages.
+
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/rosidl_defaults/blob/foxy/rosidl_default_runtime/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:160]:
```suggestion
It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rosidl_defaults/blob/foxy/rosidl_default_runtime/QUALITY_DECLARATION.md).
```

**Review by dabonnie — APPROVED** [review]

**Review by chapulina — COMMENTED** [review]
Some dependencies can be updated

File: QUALITY_DECLARATION.md [hunk 4]
```diff
@@ -1,195 +1,195 @@
...
-## Security
-
-### Vulnerability Disclosure Policy [7.i]
-
-This package does not yet have a Vulnerability Disclosure Policy.
+This document is a declaration of software quality for the `libstatistics_collector` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `libstatistics_collector` Quality Declaration
+
+The package `libstatistics_collector` claims to be in the **Quality Level 3** category.
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:5]:
```suggestion
The package `libstatistics_collector` claims to be in the **Quality Level 2** category.
```

File: QUALITY_DECLARATION.md [hunk 5]
```diff
@@ -1,195 +1,195 @@
...
+
+### Direct and Optional Runtime ROS Dependencies [5.i]/[5.ii]
+
+`libstatistics_collector` has the following runtime ROS dependencies:
+
+#### `rcl`
+
+The `rcl` package supports implementation of language specific ROS Client Libraries.
+
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/rcl/blob/foxy/rcl/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:148]: It should be level 2 as of https://github.com/ros2/rcl/pull/843

```suggestion
It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcl/blob/foxy/rcl/QUALITY_DECLARATION.md).
```

File: README.md [hunk 6]
```diff
@@ -16,4 +16,4 @@ This C++ library provides the following:
 
 ## Quality Declaration
 
-This package claims to be in the Quality Level 4 category, see the [Quality Declaration](./QUALITY_DECLARATION.md) for more details.
+This package claims to be in the Quality Level 3 category, see the [Quality Declaration](./QUALITY_DECLARATION.md) for more details.
```
Comment (chapulina) [inline README.md:19]:
```suggestion
This package claims to be in the Quality Level 2 category, see the [Quality Declaration](./QUALITY_DECLARATION.md) for more details.
```

**Review by chapulina — COMMENTED** [review]
I think this package can go straight to level 2, see the suggestions on https://github.com/ros2/rcl/pull/843

**Comment by chapulina** [comment]
@dabonnie , would you be able to merge this? Thanks!

**Pull request merged**